cli/args/flags.rs

```diff
@@ -2547,7 +2548,18 @@ Ignore linting a file by adding an ignore comment at the top of the file:
             .help("Fix any linting errors for rules that support it")
             .action(ArgAction::SetTrue),
         )
-        .arg(
+          .arg(
+            Arg::new("ext")
+                .long("ext")
+                .require_equals(true)
+                .value_name("EXT")
+                .help("Specify the file extension(s) to lint when reading from stdin.\
+                 For example, use `jsx` to lint JSX files or `tsx` for TSX files. \
+                 This argument is necessary because stdin input does not automatically infer the file type and thus requires explicit extension specification. \
+                 Example usage: `cat file.jsx | deno lint - --ext=jsx`. \
+                 Multiple extensions can be provided, separated by commas, like `jsx,tsx`."),
```

Comment:
Why would someone provide multiple file extensions? It looks like this should only allow one extension to be provided since it's an `Option<String>`?

Reply:
@dsherret 
Oh, it's just a typo in message; actually, it supports one extension.

done, I changed it and I changed  ` let ext = matches.remove_many::<String>("ext").map(|f| f.collect());`
to `let ext = matches.remove_one::<String>("ext");`